rust/ql/test/query-tests/unusedentities/unreachable.rs

```diff
@@ -203,7 +203,7 @@ fn unreachable_let_1() {
 		do_something(); // BAD: unreachable code
 	}
 
-	do_something(); // BAD: unreachable code
+	do_something();
```

Comment:
This line is unreachable, but was always something of a duplicate of the result two lines above.  I'm happy that the query no longer reports it.